src/view/configuration/dataCollectionSection.jsx

```diff
@@ -241,6 +244,9 @@ const DataCollectionSection = ({ instanceFieldName }) => {
                       value={contextOption.value}
                     >
                       {contextOption.label}
+                      {contextOption.description && (
```

Comment:
Can you use FieldDescriptionAndError component instead of Body here? That way if we ever change the styling of the descriptions, this will get the changes too.